charts/operator/values.yaml

```diff
@@ -24,7 +24,7 @@ image:
 
 configurator:
   # configurator.repository -- Repository that Redpanda configurator image is available
-  repository: docker.redpanda.com/redpandadata/configurator
+  containerRegistry: docker.redpanda.com/redpandadata/configurator
```

Comment:
This is a breaking change. Is it needed?

Reply:
I'm not that proficient in AWK to get the correct repository (container registry to be specific) from helm values yaml only if it comes from specific place. That place in values.yaml is not the same across different helm charts. The comment as far as see is different.